lcc_members_portal/controllers/custom_portal.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Lokavaluto ()
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.http import request, route
+from odoo.addons.portal.controllers.portal import CustomerPortal
+
+
+class CustomCustomerPortal(CustomerPortal):
+    @route(["/my/account"], type="http", auth="user", website=True)
+    def account(self, redirect=None, **post):
+        response = super(CustomCustomerPortal, self).account(redirect, **post)
```

Comment:
We need to test the unicity and if user.login can be different from email (ex admin account) .. think email should not be considered as only way to login .

Reply:
Check of email uniqueness already coded and delivered in PR https://github.com/Lokavaluto/lokavaluto-addons/pull/167

For the user.login, technically we still can change the login from Odoo Backend user form view (and put an non-email login), but I would prefer to not separate login and email for "standard" user, to simplify the processes and forms.

@MatSarr : do you have an opinion on that point?

Reply:
I agree with Stéph, the UX is already quite intricate, I would like it to be as smooth as possible. So I reckon the login should be the email for the user.